src/arvnetwork.h

```diff
@@ -0,0 +1,56 @@
+/* Aravis - Digital camera library
```

Comment:
Please put everything in arvnetworkprivate.h, the API here are internal to aravis and should not be exported. No need for a arvnetwork.h file.

Reply:
Now I understand; when we have accessors, there is no need to expose the internals at all.